src/test/java/seedu/pill/util/GetExpiredItemsTest.java

```diff
@@ -0,0 +1,74 @@
+package seedu.pill.util;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class GetExpiredItemsTest {
```

Comment:
Thanks for covering both successful and failure cases, as well as other edge cases. Good test coverage.